libs/sdk-ui-charts/src/highcharts/chartTypes/_chartCreators/helpers.ts

```diff
@@ -60,7 +60,7 @@ export const getHiddenSeries = (chart: Highcharts.Chart): Highcharts.Series[] =>
     chart.series && chart.series.filter((s: Highcharts.Series) => !s.visible);
 
 export const getDataPoints = (series: Highcharts.Series[]): Highcharts.Point[] =>
-    flatten(unzip(map(series, (s: Highcharts.Series) => s.points)));
+    flatten(unzip(map(series, (s: Highcharts.Series) => s.points))).filter((point) => point);
```

Comment:
Nitpick: you can use [`compact`](https://lodash.com/docs/#compact) which has the added benefit that it makes the resulting TypeScript types better (it excludes null and undefined, essentially guaranteeing there are no falsy values in the result).

Reply:
Updated!! Thanks!